extendeddatasquare.go

```diff
@@ -68,8 +71,7 @@ func (eds *ExtendedDataSquare) erasureExtendSquare(codec Codec) error {
 		return err
 	}
 
-	var shares [][]byte
-	var err error
+	errs, _ := errgroup.WithContext(context.Background())
```

Comment:
nit: Context is better to pass a param to the EDS method or constructors. In our case here, the code is unlikely to get stuck, but it is still a good practice.

Reply:
Given that the other sync.WaitGroup does not have any context as input and defaults to context.Background() I think we should keep this as is